src/plugins/vis_type_timeseries/public/services.ts

```diff
@@ -52,3 +59,15 @@ export const [getI18n, setI18n] = createGetterSetter<I18nStart>('I18n');
 export const [getChartsSetup, setChartsSetup] = createGetterSetter<ChartsPluginSetup>(
   'ChartsPluginSetup'
 );
+
+export const [getDataSourceManagementSetup, setDataSourceManagementSetup] = createGetterSetter<{
+  dataSourceManagement: DataSourceManagementPluginSetup | undefined;
+}>('DataSourceManagementSetup');
+
+export const [getHideLocalCluster, setHideLocalCluster] = createGetterSetter<{
```

Comment:
similar to my previous comment, this also can be removed

Reply:
Removed all instances of `hideLocalCluster`